Kitodo/src/main/java/org/kitodo/production/helper/Helper.java

```diff
@@ -416,12 +418,12 @@ public static Date parseDateFromFormattedString(String date) {
             return null;
         }
         try {
-            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DEFAULT_DATE_FORMAT);
             LocalDateTime localDate = LocalDateTime.parse(date, formatter);
             Instant instant = localDate.toInstant(ZoneOffset.UTC);
             return Date.from(instant);
         } catch (DateTimeParseException e) {
-            logger.info("invalid date format (yyyy-MM-dd HH:mm:ss) for date string: '" + date + "'");
+            logger.info("invalid date format ('{}') for date string: '{}'", DEFAULT_DATE_FORMAT, date);
```

Comment:
Here you change the output. I think that either () or '' is sufficient for the date format.